### PR TITLE
Fix: Fixed an issue where clicking Home on the sidebar wouldn't cancel tag searches

### LINKED ISSUE #9877: Bug: Clicking on Home option on sidebar doesnt cancel previous tag search
### Description

Search for a tag or clicking the sidebar options then click the home will keep the tag search open rather then displaying the homes content

### Steps To Reproduce

_No response_

### Expected behavior

Should go back to home page content

### Files Version

2.3.11.0

### Windows Version

10.0.19043.1889

### Relevant Assets/Logs

*

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- Closes #9877 

**Details Of Changes**
- Now clicking on Home cancel tag search (lines 1094-> 1108)
- Code Cleanup

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

File: src/Files.App/Views/ModernShellPage.xaml.cs [hunk 1]
```diff
@@ -186,7 +182,8 @@ public ModernShellPage()
             The tool has generated a custom back button in the MainWindow.xaml.cs file.
             Feel free to edit its position, behavior and use the custom back button instead.
             Read: https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/case-study-1#restoring-back-button-functionality
-            */ToolbarViewModel.BackRequested += ModernShellPage_BackNavRequested;
+            */
```
Comment (yaichenbaum) [inline src/Files.App/Views/ModernShellPage.xaml.cs:185]:
```suggestion
```

Comment (ferrariofilippo) [reply src/Files.App/Views/ModernShellPage.xaml.cs:185]: Sorry, what exactly should I change?
Should I remove the whole comment or should I change the line as `*/ToolbarViewModel...`

Comment (ferrariofilippo) [reply src/Files.App/Views/ModernShellPage.xaml.cs:185]: @yaichenbaum

**Review by yaichenbaum — COMMENTED** [review]
-

**Comment by ferrariofilippo** [comment]
It seems that also your latest comments are still pending, I'm only able to read them only through mobile app notifications or email.

**Review by yaichenbaum — APPROVED** [review]
LGTM

**Comment by yaichenbaum** [comment]
Can you resolve the merge conflicts?

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**